.github/SOLVERS.md

```diff
@@ -1,107 +1,192 @@
 
+2) <img src="https://prince.lcsb.uni.lu/jenkins/userContent/linux.png" height="20px" alt="Linux"> In a terminal window, navigate to the download directory of `Tomlab` and do the following
     ````bash
     $ chmod +x <filename>.bin
     $ sudo ./<filename>.bin
     ````
+    <img src="https://prince.lcsb.uni.lu/jenkins/userContent/windows.png" height="20px" alt="Windows"> Run the `tomlab-win64-setup_<ver>.exe` as an administrator.
+    <img src="https://prince.lcsb.uni.lu/jenkins/userContent/apple.png" height="20px" alt="macOS"> Double-click on `tomlab-osx64-setup.app`.
 
-3) Follow the installation instructions
-
-4) Copy the `tomlab.lic` license to the folder `/opt/tomlab` and change its permissions
+3) Follow the installation instructions and install `tomlab` in:
+    - <img src="https://prince.lcsb.uni.lu/jenkins/userContent/linux.png" height="20px" alt="Linux">: `/opt` such that you will have a folder `/opt/tomlab`.
+    - <img src="https://prince.lcsb.uni.lu/jenkins/userContent/windows.png" height="20px" alt="Windows">: `C:\` such that you will have a folder `C:\tomlab`.
+    - <img src="https://prince.lcsb.uni.lu/jenkins/userContent/apple.png" height="20px" alt="macOS">:
+    `/Applications` such that you will have a folder `/Applications/tomlab`
 
+4) <img src="https://prince.lcsb.uni.lu/jenkins/userContent/linux.png" height="20px" alt="Linux"> Copy the `tomlab.lic` license to the folder `/opt/tomlab` and change its permissions
     ````bash
     $ sudo chmod 644 tomlab.lic
     ````
+    <img src="https://prince.lcsb.uni.lu/jenkins/userContent/windows.png" height="20px" alt="Windows"> Copy the `tomlab.lic` license to the folder `C:\tomlab`.
+    <img src="https://prince.lcsb.uni.lu/jenkins/userContent/apple.png" height="20px" alt="macOS"> Copy the `tomlab.lic` license to the folder `/Applications/tomlab`.
 
-    If you want to confirm the correct permissions
+5) <img src="https://prince.lcsb.uni.lu/jenkins/userContent/apple.png" height="20px" alt="macOS">  <img src="https://prince.lcsb.uni.lu/jenkins/userContent/linux.png" height="20px" alt="Linux"> Set the environment variable by editing your '~/.bashrc' file:
+    ````bash
+    $ nano ~/.bashrc
     ````
-    $ stat --format '%a' <file>
+    Append the following lines (type `i` to insert text), hit the `ESC` key, then save and exit by typing `wq` and hitting ENTER:
+    ````
+    export TOMLAB_PATH="/opt/tomlab"
+    ````
+    Reload your `~/.bashrc`:
+    ````bash
+    $ source ~/.bashrc
     ````
-
 
 ## IBM ILOG CPLEX
 
 1) Download the `CPLEX` installation binary from [here](https://www-01.ibm.com/software/websphere/products/optimization/cplex-studio-community-edition/)
 
-2) Add `execute` permission to binary
-
+2) <img src="https://prince.lcsb.uni.lu/jenkins/userContent/apple.png" height="20px" alt="macOS">  <img src="https://prince.lcsb.uni.lu/jenkins/userContent/linux.png" height="20px" alt="Linux"> Add `execute` permission to binary
```

Comment:
On that website, there is a link to the community edition and the academic initiative. I agree though that this should be more visible. I will add a direct link to the academic initiative.